src/main/java/com/gotocompany/depot/http/request/util/JsonParserUtils.java

```diff
@@ -0,0 +1,115 @@
+package com.gotocompany.depot.http.request.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.JsonNodeType;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.google.gson.JsonSyntaxException;
+import com.gotocompany.depot.common.Template;
+import com.gotocompany.depot.exception.ConfigurationException;
+import com.gotocompany.depot.exception.InvalidTemplateException;
+import com.gotocompany.depot.message.ParsedMessage;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Map;
+
+
+public class JsonParserUtils {
```

Comment:
Can we add two lines javadoc documentation like why this class? what it helps for? and also couple of lines documentation for each static helper methods if the implemntation is not straightforward ?

Reply:
done